src/main/java/com/ciandt/techgallery/service/model/TechnologyActivitiesTO.java

```diff
@@ -17,29 +18,30 @@
  */
 public class TechnologyActivitiesTO {
 
-  TechGalleryUser user;
   TechGalleryUser endorserUser;
   Technology technology;
   List<TechnologyComment> comments;
   List<TechnologyRecommendation> recommendations;
   String technologyLink;
-  Date timestamp;
-  String appName;
-
-  public TechGalleryUser getUser() {
-    return user;
-  }
-
-  public void setUser(TechGalleryUser user) {
-    this.user = user;
-  }
 
+  /**
+   * Get link to view technology page according to runtime enviroment. Ex.: localhost, version-dot-.
+   * 
+   * @return link to technology page.
+   */
   public String getTechnologyLink() {
-    return technologyLink;
-  }
-
-  public void setTechnologyLink(String technologyLink) {
-    this.technologyLink = technologyLink;
+    String linkTechnology;
+    String queryString = "?id=" + technology.getId();
+    String environment = System.getProperty(Constants.RUNTIME_ENVIRONMENT_PROPERTY);
+    if (StringUtils.equals(Constants.PRODUCTION_PROPERTY, environment)) {
```

Comment:
@brunoliberal We already have a class to do that. Take a look at TechGalleryUtil